packages/typstfmt/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/astrale-sharp/typstfmt
+TERMUX_PKG_DESCRIPTION="basic formatter for the Typst language"
+TERMUX_PKG_LICENSE=Apache-2.0
+TERMUX_PKG_MAINTAINER=@termux
```

Comment:
Since I already maintain the Typst LSP, you can just put me down for maintainer on this as well.
```suggestion
TERMUX_PKG_MAINTAINER="Joshua Kahn @TomJo2000"
```

Reply:
Done.